lib/chef/knife/ec2_server_create.rb

```diff
@@ -395,6 +400,20 @@ def run
           msg_pair("Spot Request ID", spot_request.id)
           msg_pair("Spot Request Type", spot_request.request_type)
           msg_pair("Spot Price", spot_request.price)
+
+          puts 'Do you want to wait for Spot Instance Request fulfillment? (Y/N) '
+          user_choice = STDIN.gets.chomp
```

Comment:
You can replace above with https://github.com/chef/knife-ec2/blob/master/lib/chef/knife/ec2_server_delete.rb#L93